corehq/apps/accounting/utils.py

```diff
@@ -203,3 +204,20 @@ def is_accounting_admin(user):
         return user.prbac_role.has_privilege(accounting_privilege)
     except (AttributeError, UserRole.DoesNotExist):
         return False
+
+
+def get_active_reminders_by_domain_name(domain_name):
+    from corehq.apps.reminders.models import CaseReminderHandler
+    db = CaseReminderHandler.get_db()
+    key = [domain_name]
+    reminder_rules = db.view(
+        'reminders/handlers_by_reminder_type',
+        startkey=key,
+        endkey=(key + [{}]),
+        reduce=False
+    ).all()
+    return [
+        CaseReminderHandler.wrap(reminder_doc)
+        for reminder_doc in iter_docs(db, [r['id'] for r in reminder_rules])
+        if reminder_doc.get('active', True)
```

Comment:
We can also make sure here that:

reminder_doc.get('reminder_type', REMINDER_TYPE_DEFAULT) != REMINDER_TYPE_KEYWORD_INITIATED

Keyword-initiated reminders can be created in response to inbound sms so there can be a lot of them, and we don't need to worry about deactivating those because they only go out once as a response to an inbound sms and then are done.